### PR TITLE
Added feature request to invert results

### PR DESCRIPTION
**Pull request by GOAT-FARM3R**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -119,14 +120,19 @@ func (service *service) testHTTPEgress(port int) {
 		transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
 	}
 
-	timeout := time.Duration(5 * time.Second)
+	//changed this to 10 as it gave better results on the invert
+	timeout := time.Duration(10 * time.Second)
```
Comment (leonjza) [inline main.go]: Let's make a flag for timeout.

File: main.go [hunk 2]
```diff
@@ -119,14 +120,19 @@ func (service *service) testHTTPEgress(port int) {
...
 	client := http.Client{
 		Timeout:   timeout,
 		Transport: transport,
 	}
 	resp, err := client.Get(url.String())
 	if err != nil {
 		// fmt.Printf("No connection on port %d\n", port)
+		if *invertPtr{
+			fmt.Fprint(os.Stdout, "\x1b[2K")
+			fmt.Printf("[!] Looks like we have nogress using %s on port %d\n", url.String(), port)
```
Comment (leonjza) [inline main.go]: "nogress" -> "no egress"

**Review by leonjza — COMMENTED** [review]
Small changes.

**Comment by GOAT-FARM3R** [comment]
Without a second chance test I got the following:

```
===== Configuration =====
Service:	letmeout
Start Port:	20
End Port:	60000
Workers:	50
HTTPS On:	true
Ignore Certs:	false
Invert:		true
Timeout:	5
Throttle:	false
=========================

[!] Looks like we have no egress using https://go-out.letmeoutofyour.net:22 on port 22
[!] Looks like we have no egress using https://go-out.letmeoutofyour.net:202 on port 202
[!] Looks like we have no egress using https://go-out.letmeoutofyour.net:368 on port 368
[!] Looks like we have no egress using https://go-out.letmeoutofyour.net:439 on port 439
```

Whereas with a second chance test I got 

```
===== Configuration =====
Service:	letmeout
Start Port:	20
End Port:	60000
Workers:	50
HTTPS On:	true
Ignore Certs:	false
Invert:		true
Timeout:	5
Throttle:	false
=========================

[!] Looks like we have no egress using https://go-out.letmeoutofyour.net:22 on port 22
```

Port 22 was the only port blocked.

File: main.go (hunk 1 above)
Comment (GOAT-FARM3R) [reply main.go]: done

File: main.go (hunk 2 above)
Comment (GOAT-FARM3R) [reply main.go]: done

**Pull request merged**